week-07/FoxClub/day-05/Services/FoxService.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using day05.Interfaces;
+using day05.Models;
+
+namespace day05.Services
+{
+    public class FoxService : IFox
+    {
+        public List<Fox> FoxList { get; private set; }
+        public Fox CurrentFox { get; private set; }
```

Comment:
Good idea, but error prone. Because FoxService is a singleton, if a different user would login, it would change the `CurrentFox` for all other users of the app.

This "current" stuff could be placed on some kind of `SessionService` that would use a session or a cookie to store the data. In some cases this is good or even necessary. For this usecase, it's still the best to identify the "viewed" fox from the URL - it allows the best UX - saving links for later, going back and forth in browser history and so on.